src/Item/Item.schema.ts

```diff
@@ -0,0 +1,47 @@
+import { metricsSchema } from '../Metrics/Metrics.types'
+import { ItemRarity, ItemType } from './Item.types'
+import { wearableSchema } from './wearable/types'
+
+export const itemSchema = Object.freeze({
```

Comment:
Question: Why are we splitting this schema into its own file here but not in the other models?

Reply:
I'm not opposed to that, mostly if it's a circular dep concern!
Still, I think it's valid to say that an schema can live in `XX.types.ts` if we look at that file as the description of how looks. Should we add an issue so we can track it?

Reply:
Done! https://app.zenhub.com/workspaces/dapps-5ffc44ecec9f8500140e173c/issues/decentraland/builder-server/298